lib/src/models/community_feed.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:lightning_api/src/models/models.dart';
+
+part 'community_feed.g.dart';
+
+@JsonSerializable(explicitToJson: true)
+class CommunityFeed extends Equatable {
+  const CommunityFeed({
+    required this.posts,
+    required this.lastOpIndices,
+    required this.oldestQuery,
+  });
+
+  factory CommunityFeed.fromJson(Map<String, dynamic> json) {
+    return _$CommunityFeedFromJson(json);
+  }
+
+  @JsonKey(
+    fromJson: Authorperm.deserializeList,
+    toJson: Authorperm.serializeList,
+  )
+  final List<Authorperm> posts;
+  final Map<String, int> lastOpIndices;
+  final DateTime oldestQuery;
```

Comment:
This too, I'm not sure what "oldest query" means (I did see the comment about it not being globally oldest, but my question is more about what does even "query" mean?) A query is a manual search from a user, and so I don't understand why something like that would be in a result which doesn't have to do with search.

Reply:
I can change it to oldestOperation, if that's ok. Not sure how I can convey the meaning, but operation fits better than query

Reply:
I still don't understand "operation"; putting a JSON in the samples directory would help clarify what the API wants to convey